CMakeLists.txt

```diff
@@ -32,7 +32,6 @@ list(INSERT CMAKE_MODULE_PATH 0 "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 option(CREATE_SYMLINKS "Create symlinks to javascript modules and auxillary files - for development purposes" OFF)
 option(CMAKE_RUN_CLANG_TIDY "Run clang-tidy" OFF)
 option(BUILD_TESTING "Run unit tests" OFF)
-option(ISO15118_2_GENERATE_AND_INSTALL_CERTIFICATES "Automatically generate and install certificates for development purposes" ON)
```

Comment:
It looks like there is no way to turn off certificate generation, but I think one should still be able to provide their own certificates if they so choose

Reply:
Correct, this needs to be discussed - therefor I didn't at it yet

Reply:
I agree certificate generation should be an option